RFS/src/main/java/com/rfs/tracing/BaseRootRfsContext.java

```diff
@@ -0,0 +1,19 @@
+package com.rfs.tracing;
+
+import io.opentelemetry.api.OpenTelemetry;
+import org.opensearch.migrations.tracing.IContextTracker;
+import org.opensearch.migrations.tracing.RootOtelContext;
+
+public class BaseRootRfsContext extends RootOtelContext {
```

Comment:
We've got many stand alone apps that are all under the migration space, what about MigrationAppContext? If we are following the model that everything should be shared, shouldn't this migrate out into the core utils space?